assets/script.js

```diff
@@ -196,13 +200,17 @@ function getArtistInfo(userInput) {
             } else {
               // Appending the album
               $("#albums>ul").append(
-                '<li><a id="albums" class="waves-effect waves-light modal-trigger" href="#album-modal"><img src="' +
+                '<li><a id="albums" class="image waves-effect waves-light modal-trigger" href="#album-modal"><img class="materialboxed" src="' +
```

Comment:
```suggestion
                '<li><a class="image waves-effect waves-light modal-trigger" href="#album-modal"><img class="materialboxed" src="' +
```

Reply:
Also make the class name more descriptive. What image? there's loads of images on the site, "I've just joined the team and I don't know what image you mean" make your code accessible to others